### PR TITLE
Shrestha2020distribution branch

### PR DESCRIPTION
**Pull request by WenfeiXiao**
Updated shrestha2020distribution-extraction.md and raw data.

### CODE REVIEW

**Review by YWAN446 — COMMENTED** [review]
Did some cleaning for the shrestha2020distribution-extraction.md. The changes of arts and kim are carried over, which are the same for the pr #45.

File: data/shrestha2020distribution/shrestha2020distribution-extraction.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+```python
+shrestha2020distribution = dict(title="Distribution of Transmission Potential During Nonsevere COVID-19 Illness",
+        doi="10.1093/cid/ciaa886",
+        description=folded_str('This study evaluated the transmission potential of COVID-19 by examining viral load over time. Over six weeks, 230 healthcare personnel underwent 528 tests at the Cleveland Clinic. Cycle threshold (Ct) values were obtained using RT-PCR targeting the N gene, and viral loads were calculated. Data were obtained from the combined dataset in the supplementary materials of Challenger et al. BMC Medicine (2022) 20:25 (https://doi.org/10.1186/s12916-021-02220-0).\n'),
+        analytes=dict(swab_SARSCoV2_N=dict(description=folded_str("Cycle threshold (Ct) values were quantified using RT-PCR targeting the N gene in nasopharyngeal swab samples. Ct values were then converted into copies per mL, with a detection limit of 20 copies/mL. Viral load calculations were based on the minimum detectable viral load (MDVL) and Ct values.\n"),
+                                    limit_of_quantification="unknown",
+                                    limit_of_detection=20,
```
Comment (YWAN446) [inline data/shrestha2020distribution/shrestha2020distribution-extraction.md:64]: @WenfeiXiao Hi Fiona, I found out the limit of detection in the paper is 20 gc/uL for PCR. I did some back-calculation based on:
```
To maintain consistency, a decision was made to test all HCP specimens on the same testing platform. The reverse-transcription (RT)–PCR test developed at the CDC was used, and internal validation studies were performed in-house before introduction of the test. Using commercially available plasmids that contained the nucleocapsid (N) gene, the limit of detection was found to be 20 copies/µL for upper respiratory specimens and 2 copies/µL for lower respiratory specimens.

For each specimen, 200 µL of clinical specimen in universal/viral transport medium was rendered noninfectious within a biological safety cabinet through the addition of 200 µL of Bacterial Lysis Buffer (Roche Diagnostics, Indianapolis, IN). Nucleic acid extraction was performed using 200 µL of the inactivated specimen using the MagNA Pure system (Roche). Five microliters of eluate was added to 15 µL of PCR mastermix for each of the PCR reactions. The test was designed to target 3 separate regions of the virus nucleocapsid (N) gene. An amplification control targeting a human gene, the RNase P gene, was assessed for each specimen. Thus, each test consisted of 4 separate PCRs. Testing was performed on an ABI 7500 or ABI 7500 Fast Dx (ThermoFisher, Waltham, MA). All 3 PCRs for N gene targets (ie, N1, N2, and N3) had to be positive for a specimen to be reported as positive. If only 1 or 2 of the targets generated amplification, then the specimen was considered indeterminate and repeated either with the CDC assay or the Simplexa COVID-19 Direct Kit (DiaSorin, Cypress, CA).
```

and got the LOD to be 3000 gc/mL, which does not make sense. The minimum of the value in Challenger's dataset for this study seems to be around 400 gc/mL. I will suggest to change this `20` to `unknown` as we can not decide the LOD for raw sample.

File: data/shrestha2020distribution/shrestha2020distribution-extraction.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    participants.append(participant)
+```
+
+The data is formatted and output as a YAML file.
+
+```python
+shrestha2020distribution = dict(title="Distribution of Transmission Potential During Nonsevere COVID-19 Illness",
+        doi="10.1093/cid/ciaa886",
+        description=folded_str('This study evaluated the transmission potential of COVID-19 by examining viral load over time. Over six weeks, 230 healthcare personnel underwent 528 tests at the Cleveland Clinic. Cycle threshold (Ct) values were obtained using RT-PCR targeting the N gene, and viral loads were calculated. Data were obtained from the combined dataset in the supplementary materials of Challenger et al. BMC Medicine (2022) 20:25 (https://doi.org/10.1186/s12916-021-02220-0).\n'),
+        analytes=dict(swab_SARSCoV2_N=dict(description=folded_str("Cycle threshold (Ct) values were quantified using RT-PCR targeting the N gene in nasopharyngeal swab samples. Ct values were then converted into copies per mL, with a detection limit of 20 copies/mL. Viral load calculations were based on the minimum detectable viral load (MDVL) and Ct values.\n"),
```
Comment (YWAN446) [inline data/shrestha2020distribution/shrestha2020distribution-extraction.md:62]: Change `20 copies/mL` to `20 copies/uL` amd add a sentence to explain we can not determine the LOD for raw samples.

**Review by YWAN446 — COMMENTED** [review]
A comment about the LOD unit error.

File: data/shrestha2020distribution/shrestha2020distribution-extraction.md (hunk 2 above)
Comment (tillahoffmann) [reply data/shrestha2020distribution/shrestha2020distribution-extraction.md]: What do you think about converting everything from `uL` to `mL` so we have consistent units across all the datasets?

**Review by YWAN446 — APPROVED** [review]

**Pull request merged**